litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/AddExperimentModal.tsx

```diff
@@ -70,28 +83,42 @@ const AddExperimentModal: React.FC<AddExperimentModalProps> = ({
           {t('createWorkflow.tuneWorkflow.selectAvailableExp')} {hubName} .{' '}
           {t('createWorkflow.tuneWorkflow.afterSelect')}
         </Typography>
-        <FormControl variant="outlined" className={classes.formControl}>
-          <InputLabel id="demo-simple-select-label" className={classes.label}>
-            {t('createWorkflow.tuneWorkflow.selectExp')}
-          </InputLabel>
-          <Select
-            labelId="demo-simple-select-label"
-            value={selectedExp}
-            onChange={onSelectChange}
-            label="Cluster Status"
-            MenuProps={MenuProps}
-            input={<OutlinedInput labelWidth={150} />}
-          >
-            {allExperiments.map((exp: ChartName) => (
-              <MenuItem
+        <br />
+        <Search
+          data-cy="agentSearch"
+          id="input-with-icon-textfield"
+          placeholder="Search"
+          value={search}
+          onChange={(event) => setSearch(event.target.value)}
+        />
+
+        <br />
+        <br />
```

Comment:
can the below div be given a padding or. margin instead of two times <br>.. single also would have been ok.

Reply:
Sure.